source/std.tex

```diff
@@ -72,6 +72,7 @@
 %%--------------------------------------------------
 %% add special hyphenation rules
 \hyphenation{tem-plate ex-am-ple in-put-it-er-a-tor name-space name-spaces}
+\hyphenation{non-ze-ro}
```

Comment:
a-c done. Note I found "non-ze-ro" in the online Merriam-Webster, but non-zero is good enough for our purposes, I'd guess.